source/code/plugins/in_sudo_tail.rb

```diff
@@ -58,7 +58,7 @@ def configure(conf)
  
       @parser = Plugin.new_parser(conf['format'])
       @parser.configure(conf)
-
+      @path.gsub!("," , " ")       #in case multiple paths are separated by commas
```

Comment:
I've updated changes to log each filename for the case when path includes *
I am currently assuming that in case of multiple paths, they are separated by comma and the file path does not contain commas (as in the original tail plugin). We could change the "path" field in this input plugin to an array instead of a string like so: path ["/var/log/messages*", "/var/log/audit/audit.log"]